components/VersionHistory/VersionHistory.vue

```diff
@@ -1,21 +1,23 @@
 <template>
   <div class="version-history-container">
     <large-modal
+      v-if="dialogVisible"
       :visible="dialogVisible"
```

Comment:
Do you need both of these?

Reply:
@hsorby  yes, some of the contents in the large-modal are not available onMount. 

using v-if allows them to only be loaded just in time.

the `:visible` one is required for the modal so we end up needing both

Reply:
I could remove the `v-if`
I would put all of the logic in `computed` properties to check if the object parameters exist or not.
ie:
```
  <p>Published on: {{logPublishedAt}}</p>

//...
  computed: {
    logPublishedAt: function(){
      if( this.changeLogFileInfo.versionPublishedAt) {
        return formatDate(this.changeLogFileInfo.versionPublishedAt)
      }
      return ''
    }
```


I think the `v-if` is cleaner personally, but I could use your feedback so I can get this in before the weekend

Reply:
Using a computed property is the way that I would do it personally. I think that is a very Vue way of handling this problem.

Reply:
@hsorby I have switched to the way you would like to do it. All good for approval now?